Content.Shared/Cuffs/SharedCuffableSystem.cs

```diff
@@ -434,6 +437,49 @@ private void UpdateHeldItems(EntityUid uid, EntityUid handcuff, CuffableComponen
                 EnsureComp<UnremoveableComponent>(virtItem2.Value);
         }
 
+        /// <summary>
+        /// Checks if the handcuff is stackable, and creates a new handcuff entity if the stack requires it.                                   
+        /// </summary>
+        /// <returns></returns>
+        public bool TrySpawnCuffSplitStack(EntityUid handcuff, EntityUid user, EntityUid target, [NotNullWhen(true)] out EntityUid? handcuffsplit)
+        {
+            if (!HasComp<HandcuffComponent>(handcuff))
+            {
+                handcuffsplit = null;
+                return false;
+            }
+            if (TryComp<StackComponent>(handcuff, out var stackComp))
+            {
+                if (_stacks.GetCount(handcuff, stackComp) >= 1)
+                {
+                    _stacks.Use(handcuff, 1, stackComp);
+
+                    if (_net.IsServer) /// let the server spawn because client mispredicts
+                    {
+                        var pos = Transform(target).Coordinates;
+                        handcuffsplit = Spawn("Zipties", pos); /// This should somehow get the proto ID instead of zipties, but fuck if I know how.
```

Comment:
Meta(target).Prototype or something

also why the fuck are you duplicating stack code!!!!

Reply:
this is 1-for-1 copied code from IProduceWidgets (PR 21349 linked above), debugged and updated

this is the best i can do, given i know minimal C# but i'll see if i can get someone to help me improve it

Reply:
should be possible to just make zipties stackable up to 1 then have a lingering "stack" that regenerates or something

Reply:
do you mean just making the ziptie stack & stacktype without the change to the c# changes?

Reply:
yea

Reply:
finally got around to testing this, removing this doesn't work :(